static/js/containers/LearnerSearchPage_test.js

```diff
@@ -153,6 +159,16 @@ describe('LearnerSearchPage', function () {
     });
   });
 
+  it('does not render the email link for learner users', () => {
+    const EMAIL_LINK_SELECTOR = '#email-selected';
```

Comment:
Can you move this into the parent `describe` and remove the other `EMAIL_LINK_SELECTOR` variables from the other tests?